experiment_management/bundle_integrate_mlflow.ipynb

```diff
@@ -0,0 +1,272 @@
+   "source": [
+    "resource = \"https://msd-for-monai.s3-us-west-2.amazonaws.com/Task09_Spleen.tar\"\n",
+    "md5 = \"410d4a301da4e5b2f6f86ec3ddba524e\"\n",
+    "\n",
+    "compressed_file = os.path.join(root_dir, \"Task09_Spleen.tar\")\n",
+    "data_dir = os.path.join(root_dir, \"Task09_Spleen\")\n",
+    "print(data_dir)\n",
+    "if not os.path.exists(data_dir):\n",
+    "    download_and_extract(resource, compressed_file, root_dir, md5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use MLflow in MONAI bundle\n",
+    "\n",
+    "In this part, we will take the [spleen segmentation bundle](https://github.com/Project-MONAI/model-zoo/tree/dev/models/spleen_ct_segmentation) as an example to show how to enable MLflow in it. Typically there are two ways to enable MLflow in a bundle training process. The easiest way is to add `--tracking \"mlflow\"` at the end of the command line. Some extra parameters like `tracking_uri` and `experiment_name` can also be added this way. The second is to add a config json file as input. In this file, users can define their own setting on MLflow. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download spleen segmentation bundle"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "monai.bundle.download(name=\"spleen_ct_segmentation\", version=\"0.3.7\", bundle_dir=\"./\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run spleen bundle with MLflow parameter"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The command line in the next cell is the easiest way to run the spleen segementation training  bundle with mlflow. Please modify the `--dataset_dir` with your own path of dataset. The parameter `--tracking \"mlflow\"` at the end of the original command is to enable the mlflow during training. Parameters `--tracking_uri`, `--experiment_name` and `--run_name` can also be added and modified to change the tracking uri, experiment name and run name of mlflow. To enable MLflow in multi-gpu training is as the same with single gpu by adding `--tracking \"mlflow\"` to the end of command line. \n",
+    "\n",
+    "A `mlruns` folder will be created in the `spleen_ct_segmentation/eval` folder during the running. Running the command `mlflow ui` in this folder can set a webpage UI for tracking. By default, the address will be `http://127.0.0.1:5000`. If there is a confliction of port or host address, `--port` and `--host` parameters can be modified to new one. \n",
+    "Here is the tracking result.\n",
+    "![image](./extra_pics/mlflow_default_result.png)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!cd spleen_ct_segmentation;python -m monai.bundle run training \\\n",
```

Comment:
I think no need the `cd spleen_ct_segmentation`.

Thanks.

Reply:
Hi @Nic-Ma ,
    Currently, this tutorial will download the spleen_ct_segmentation and extract it as the `spleen_ct_segmentation` folder in the same folder level as the tutorial. If removing the `cd spleen_ct_segmentation` folder, the python command will be executed in the wrong folder and report `cannot find configs/metadata.json file` error. What would you suggest to avoid this error, if removing the `cd` command line?

Thanks,
Bin